migrations/1_create_users_projects_files.js

```diff
@@ -8,6 +8,10 @@ exports.up = function (knex) {
       t.increments('id');
       t.integer('user_id').notNullable().references('id').inTable('users');
       t.text('title').notNullable();
+      t.text('tags');
```

Comment:
maybe we want a separate table for this? That might make things even easier when the Thimble gallery is implemented (for e.g. filtering makes by tags)?